exchanges/btcmarkets/btcmarkets.go

```diff
@@ -218,56 +199,45 @@ func (b *BTCMarkets) GetMarketCandles(ctx context.Context, marketID, timeWindow
 func (b *BTCMarkets) GetTickers(ctx context.Context, marketIDs currency.Pairs) ([]Ticker, error) {
 	var tickers []Ticker
 	params := url.Values{}
+	pFmt, err := b.GetPairFormat(asset.Spot, true)
+	if err != nil {
+		return nil, err
+	}
 	for x := range marketIDs {
-		params.Add("marketId", marketIDs[x].String())
+		params.Add("marketId", marketIDs[x].Format(pFmt).String())
```

Comment:
🐒 
```suggestion
		params.Add("marketId", pFmt.Format(marketIDs[x]))
```
This appears to be the only occurrence of this nit:
```
git diff -p upstream/master...HEAD | rg "\.Format\(.*String\("
```